lib/kafka-shared/Makefile

```diff
@@ -0,0 +1,40 @@
+# cert files (in directory created by `certs` target)
+KAFKA_CA_CERT := $(shell pwd)/certs-0/ca.crt
+KAFKA_CLIENT_KEY := $(shell pwd)/certs-0/tls.key
+KAFKA_CLIENT_CERT := $(shell pwd)/certs-0/tls.crt
+
+CONTEXT := $(shell basename $(shell dirname ${PWD}))
+ENV := $(shell echo ${CONTEXT} | cut -d"-" -f1)
+PROVIDER := $(shell echo ${CONTEXT} | rev | cut -d"-" -f1 | rev)
+
+certs:
+	# obtain the certificates from the kafka pod
+	kubectl exec --context "${CONTEXT}" -n "pubsub" "kafka-shared-0" -- tar cfh - "certs-0/ca.crt" "certs-0/tls.key" "certs-0/tls.crt" | tar xf -
+
+init: certs
+	# terraform init
+	terraform init \
+  -backend-config region=eu-west-1 \
+  -backend-config bucket=uw-${ENV}-pubsub-tf-applier-state \
+  -backend-config key="${CONTEXT}"/kafka-shared \
+  -backend-config encrypt=true \
+  -reconfigure \
+  -upgrade
```

Comment:
pedantic: inconsistent spacing (mix of tabs+spaces?)

Reply:
It is mix of tabs and spaces indeed. My IDE [makefile plugin](https://plugins.jetbrains.com/plugin/9333-makefile-language) does this auto- formatting for me. After a few minutes of trying to find the information about correct Makefile syntax in GNU docs I decided I will just trust it :monkey:

Reply:
> It is mix of tabs and spaces indeed. 

Can we just stick with one? Tabs will render differently on different setups (e.g. will it be 2 spaces, or 4). In particularly it looks odd on github (where I think tabs render as 8 spaces)

> After a few minutes of trying to find the information about correct Makefile syntax in GNU docs I decided I will just trust it 

The syntax demands (https://www.gnu.org/software/make/manual/make.html#Rule-Syntax):

> The recipe lines start with a tab character (or the first character in the value of the .RECIPEPREFIX variable; see [Other Special Variables](https://www.gnu.org/software/make/manual/make.html#Special-Variables)).

So the first line _has_ to be a tab, so the others might as well be too

Reply:
[Applied](https://github.com/utilitywarehouse/kafka-cluster-config/pull/116/commits/de8e3f817eaac547bc6f8b550031df75c1f1f26f)